RealmSwift/SwiftUI.swift

```diff
@@ -471,6 +526,14 @@ extension Binding where Value: ObjectBase & ThreadConfined {
     public subscript<V>(dynamicMember member: ReferenceWritableKeyPath<Value, V>) -> Binding<V> where V: _Persistable {
         createBinding(wrappedValue, forKeyPath: member)
     }
+    /// :nodoc:
+    public subscript<V>(dynamicMember member: ReferenceWritableKeyPath<Value, V>) -> Binding<V> where V: _Persistable & RLMSwiftCollectionBase & ThreadConfined {
```

Comment:
I thought that compounding protocol conformance like this was the undesired way to do it? and instead should be like:
```swift
where V: _Persistable, V: RLMSwiftCollectionBase, V: RLMSwiftCollectionBase ...
```